src/commands/config.ts

```diff
@@ -45,6 +45,8 @@ export default class ConfigCommand implements ICommand {
 				streamShoutString = `${shoutChannel?.name || 'ERR-404'} (${ss.streamShout})`;
 			}
 
+			let streamTimeoutString = ss.streamTimeout + " hours";
```

Comment:
Should return `Off` if set to 0. To be more inline with the rest of the settings